### PR TITLE
See individual reminders

### LINKED ISSUE #3: See Individual Reminder
When the user clicks on one of the reminders on the page, they are taken to an individual reminders route (ie: '/reminders/1', if the id of the reminder is `1`). The title of the reminder object should be displayed.

So, what needs to happen here?

* You'll need a `reminder` route in addition to the `reminders` route.
* It should be nested inside the `reminders` route.
* The user should still see all of the reminders from your previous feature. The specific reminder they are looking at now should be rendered in the outlet of the `reminders.hbs` template.
* There should be some CSS for the `.active` class.

There is a failing test set up in `tests/acceptance/reminder-list-test.js` to get you started - this test needs to be rewritten to reflect the route you are expecting to visit.

### PR DESCRIPTION
**Pull request by dshendrickson**
Purpose

When the user clicks on one of the reminders on the page, they are taken to an individual reminders route (ie: '/reminders/1', if the id of the reminder is 1). The title of the reminder object should be displayed.

Issue 3

Approach

This pull request addresses all of the following items:

You'll need a reminder route in addition to the reminders route.
It should be nested inside the reminders route.
The user should still see all of the reminders from your previous feature. The specific reminder they are looking at now should be rendered in the outlet of the reminders.hbs template.
There should be some CSS for the .active class.
Learning

Worked through documentation on the react.js site, specifically:
Nested Routes

Test coverage

Tests modified as per specifications.

Issue:

closes #3

### CODE REVIEW

File: tests/acceptance/reminder-list-test.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ test('viewing the homepage', function(assert) {
   visit('/');
 
   andThen(function() {
-    assert.equal(currentURL(), '/reminders');
+    assert.equal(currentURL(), '/');
```
Comment (Tman22) [inline tests/acceptance/reminder-list-test.js]: Shouldn't this be '/reminders'? In the next test, after clicking on a single reminder you should be directed to '/reminders/1'

Comment (nfosterky) [inline tests/acceptance/reminder-list-test.js]: Does this test pass? `'/'` should redirect to `'/reminders'`

**Comment by nfosterky** [comment]
What's the status on this?

File: tests/acceptance/reminder-list-test.js (hunk 1 above)
Comment (andrewLcrist) [inline tests/acceptance/reminder-list-test.js]: We thought the route example was merely an example, but we've updated the route to `/reminders/1`

**Comment by andrewLcrist** [comment]
Requested changes have been made.

File: tests/unit/routes/reminders/reminder-test.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
```
Comment (martensonbj) [inline tests/unit/routes/reminders/reminder-test.js]: Probably won't need this test - the acceptance tests routing to the various paths handle testing the route. You can delete it.

File: tests/acceptance/reminder-list-test.js [hunk 3]
```diff
@@ -25,7 +25,7 @@ test('clicking on an individual item', function(assert) {
   click('.spec-reminder-item:first');
 
   andThen(function() {
-    assert.equal(currentURL(), '/1');
+    assert.equal(currentURL(), '/reminders/1');
     assert.equal(Ember.$('.spec-reminder-item:first').text().trim(), Ember.$('.spec-reminder-title').text().trim());
```
Comment (martensonbj) [inline tests/acceptance/reminder-list-test.js]: The title of this file is `reminder-list-test` . Either change the name of the file to more accurately describe what you're testing (aka `reminders.test.js`) or create a new file for the individual reminders test.

File: tests/acceptance/reminder-list-test.js (hunk 1 above)
Comment (martensonbj) [inline tests/acceptance/reminder-list-test.js]: It looks like there's confusion on what routes are being asked for (which is partly our fault for initially requesting a '/' route.) Users should be redirected to `'/reminders'` when they visit '/'. Please write a test to reflect this and make sure it's passing.

File: app/templates/reminders/reminder.hbs [hunk 4]
```diff
@@ -0,0 +1,5 @@
+
+<h3 class="spec-reminder-title">{{model.title}}</h3>
+<body>{{model.notes}}</body>
```
Comment (martensonbj) [inline app/templates/reminders/reminder.hbs]: There should only be one set of `<body>` HTML tags per document, which already exist for this project in the `index.html` file. A better semantic tag here would be `<section>` or a `<div>` with a particular class.

File: app/styles/app.css [hunk 5]
```diff
@@ -0,0 +1,3 @@
+.active{
+  background: blue;
```
Comment (martensonbj) [inline app/styles/app.css]: Strong color choice 😉

File: app/router.js [hunk 6]
```diff
@@ -7,7 +7,9 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
-  this.route('reminders');
+  this.route('reminders', {path: '/'}, function() {
```
Comment (martensonbj) [inline app/router.js]: I commented on this in your test file also but we actually want the route to display `'/reminders'`. Sorry for any confusion here - please make that change and update your test to reflect the route users should see on app-load.

**Comment by martensonbj** [comment]
Also per convention lets use classic sentence case for commit messages instead of all caps.

**Comment by andrewLcrist** [comment]
Requested changes made.

**Comment by andrewLcrist** [comment]
Requested changes made.

Also, per the article included in class, it says to capitalize the first line? Unless it's just a misinterpretation. http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html

**Comment by martensonbj** [comment]
Way to 💪 through. We'll tackle the other change in upcoming PRs.

**Pull request merged**